src/mon/FSCommands.h

```diff
@@ -59,9 +59,17 @@ class FileSystemCommandHandler : protected CommandHandler
   virtual ~FileSystemCommandHandler()
   {}
 
-  bool can_handle(std::string const &prefix_)
+  int is_op_allowed(MonOpRequestRef op, FSMap &fsmap, const cmdmap_t& cmdmap,
+		    std::stringstream &ss);
+
+  int can_handle(std::string const &prefix_, MonOpRequestRef op, FSMap &fsmap,
+	          const cmdmap_t& cmdmap, std::stringstream &ss)
   {
-    return get_prefix() == prefix_;
+    if (get_prefix() != prefix_) {
+      return 1;
```

Comment:
```suggestion
      return 0;
```

Reply:
Return `1` if handleable.

Reply:
Fixed it. GitHub doesn't mark this comment outdated because coinidentally there's a different `return 1` on the same line number.